1.8/usage/registry.md

```diff
@@ -0,0 +1,167 @@
+1. __Self-signed Certificate__ This option provides decent security and is more manageable.
+1. __Insecure__ This option should be avoided when possible. It may even conflict with Docker-in-Docker containers such as Jenkins.
+
+For more information on setting up a Docker Trusted registry see [deploying a registry server](https://docs.docker.com/registry/deploying/) via the Docker docs.
+
+## Naming and discovery options
+
+After the registry is deployed clients will need to access it. When you create the certificate, you should consider the naming and discovery approach used to access the service.  You can use any of the [service discovery](/docs/1.8/usage/service-discovery/) options available in DC/OS:
+
+- __Mesos-DNS__ with host port mapping is an easy option, but requires that ports are managed as resources in the cluster.
+- __Minuteman__ provides a stable endpoint and you can use either the VIP or a DNS entry that references the VIP. A DNS entry can add a bit more complexity but improves consumption of the service.
+- __Marathon-LB__ is another alternative that will work, but is better suited for north-south ingress traffic.
+
+Note: if you are using a port other than `443`, the port will have to be specified when the service is accessed.
+
+All of these options work fine, however in this tutorial we focus on using a self-signed certificate via VIPs (Minuteman). For example VIP_0: registry:5000.
+
+## Deploy a Docker registry using a self-signed certificate
+
+At a high level, here are the three steps required to set up a private Docker registry:
+
+1. Create a self-signed certificate with the correct common name and subject alternative names.
+1. Configure the Docker Registry service with the public and private key for the certificate.
+1. Configure Docker daemons to trust the self-signed certificates.
+
+Open an SSH connection to a master node in the cluster with credentials forwarding enabled (option `-A`), so that an SSH connection can be made to other nodes in the cluster. Note: this tutorial uses [jq](https://stedolan.github.io/jq/), but you can also carry out the steps using, for example, `grep`.
+
+Apply this workaround for minuteman to all of your agents so that you can push and pull images to and from the registry.
+
+```bash
+$ for i in $(curl -sS master.mesos:5050/slaves | jq '.slaves[] | .hostname' | tr -d '"'); do ssh "$i" -oStrictHostKeyChecking=no "sudo sysctl -w net.netfilter.nf_conntrack_tcp_be_liberal=1"; done
+```
+
+### Create a self signed cert
+
+The first step is to create a self-signed certificate to secure a Docker registry. It’s important that the common name used in the certificate matches the domain name that is used to access the registry.
+
+You can create a self-signed certificate to secure a registry by using OpenSSL. This command creates a certificate that uses a minuteman VIP to access the registry. 
```

Comment:
Are there other types of DC/OS VIPs, besides minutemen? If not, change to `...uses a VIP to access...`. If so, change to `...uses a Minuteman VIP to access...`